core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1477,19 +1477,19 @@ public MetricKey build() {
           .build();
   public static final MetricKey TOTAL_EXTRA_TIME =
        new Builder("JvmPauseMonitor.totalExtraTime")
-           .setDescription("Total extra sleep time.")
+           .setDescription("The total time of jvm thread sleep.")
            .setMetricType(MetricType.GAUGE)
            .setIsClusterAggregated(false)
            .build();
   public static final MetricKey INFO_TIME_EXCEEDED =
        new Builder("JvmPauseMonitor.infoTimeExceeded")
-           .setDescription("Times extra sleep time exceed INFO.")
+           .setDescription("The times sleep time exceed mInfoThreshold , need to record a info level log.")
```

Comment:
Users won't know what's mInfoThreshold, please change to code independent description